.travis.yml

```diff
@@ -23,11 +24,19 @@ before_script:
 - go install
 - $(exit $(go fmt $EXCLUDE_VENDOR | wc -l))
 - go vet $EXCLUDE_VENDOR
-- $(exit $(misspell -locale US . | grep -v "vendor/" | wc -l))
+- $(exit $(misspell -locale US $EXCLUDE_VENDOR_AND_PROTO_DIR | wc -l))
+- $(exit $(misspell -locale US README.md | wc -l))
 - staticcheck $EXCLUDE_VENDOR_AND_PROTO_DIR
 script:
 - mysql -u root -e "CREATE USER 'nss'@'localhost' IDENTIFIED BY 'password'; GRANT ALL PRIVILEGES ON *.* TO 'nss'@'localhost'; CREATE DATABASE test_nats_streaming;"
 - go test -i $EXCLUDE_VENDOR
 - if [[ "$TRAVIS_GO_VERSION" =~ 1.12 ]]; then ./scripts/cov.sh TRAVIS; else go test -failfast $EXCLUDE_VENDOR; fi
-after_success:
-- if [[ "$TRAVIS_GO_VERSION" =~ 1.11 ]] && [ "$TRAVIS_TAG" != "" ]; then ./scripts/cross_compile.sh $TRAVIS_TAG; ghr --owner nats-io --token $GITHUB_TOKEN --draft --replace $TRAVIS_TAG pkg/; fi
+
+deploy:
+  provider: script
+  skip_cleanup: true
+  script: curl -sL http://git.io/goreleaser | bash
+  verbose: true
+  on:
+    tags: true
+    condition: $TRAVIS_GO_VERSION =~ 1.11
```

Comment:
Not upgrading to 1.12 here?

Reply:
With streaming and file, the sync on macOS perf degradation from 1.11 to 1.12 is significant. I will ultimately..

Reply:
Ah yes, got it.